monix-eval/shared/src/main/scala/monix/eval/TaskCircuitBreaker.scala

```diff
@@ -535,9 +536,9 @@ object TaskCircuitBreaker {
       * when the `Open` state is to transition to [[HalfOpen]].
       *
       * It is calculated as:
-      * {{{
+      * ```scala
```

Comment:
Does this Github-flavored markdown annotation work?

Reply:
sbt-doctest ignores it, if that's what you ask
i did that as the code refers to fields, and is not compilable as is

To me this particular scaladoc example is not really an example, but implementation explanation
to make it compilable we will have to write smth like
```scala
  val open = Open(1234, 5 seconds)
  require(open.startedAt + open.resetTimeout.toMillis == open.expiresAt)
```
but not sure if it's helpful or any better
wdyt?

Reply:
It’s indeed not a sample. But we have to have ScalaDoc compile it without producing garbage and I don’t think this does.

One thing we can do is to do that inline via single back-ticks.

Reply:
you mean 
```
|`startedAt + resetTimeout.toMillis`

instead of 

|```scala
|startedAt + resetTimeout.toMillis
|```
```
?

Reply:
Yes, that can work.

Reply:
ok, will address that in PR for Task